tests/unit/scripts/test_lint.py

```diff
@@ -237,6 +237,12 @@ def test_check_exclusions(create_testable_file, exclusion):
         assert item not in result.out
 
 
+def test_check_cylc_file_jinja2_comments(create_testable_file):
+    # Repalce the '# {{' line to be '{# {{' which should not be a warning
+    result, _ = create_testable_file(LINT_TEST_FILE.replace('# {{', '{# {{'), ['style'])
```

Comment:
It's potentially fragile, I think simply this will do
```suggestion
    result, _ = create_testable_file('{# {{ foo }} #}', ['style'])
```

Reply:
Thanks, very good point. Adopted.